app/src/main/java/com/example/arc_exapmle/AddNoteKtActivity.kt

```diff
@@ -61,9 +62,10 @@ class AddNoteKtActivity : AppCompatActivity() {
 
         val note = NoteEntity(title, description, priority)
 
+
         val data = Intent()
 
-        data.putExtra(EXTRA_NOTE, note)
+        data.putExtra(EXTRA_NOTE, NoteUI(note.getId(), note.title, note.description, note.priority))
         setResult(Activity.RESULT_OK, data)
```

Comment:
mitoni bejaye activityResult
Note ro bezari to DB va DB khodesh baghie safahat ro khabar mikone